substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/io/Utf8.java

```diff
@@ -36,8 +36,22 @@ public final class Utf8 {
     private Utf8() {
     }
 
+    private static int utf8Size(char c) {
+        // Based On
+        // https://github.com/openjdk/jdk21u/blob/jdk-21.0.4%2B7/src/hotspot/share/utilities/utf8.cpp#L409-L418
```

Comment:
```suggestion
    // @BasedOnJDKFile("https://github.com/openjdk/jdk21u/blob/jdk-21.0.4%2B7/src/hotspot/share/utilities/utf8.cpp#L409-L418")
    private static int utf8Size(char c) {
```

Reply:
Same justification as in https://github.com/graalvm/graalvm-community-jdk21u/pull/60#discussion_r2074027575

Reply:
OK.